### DISCUSSION TITLE
Can firejail stop spywares in PDF files from scam emails?

### DISCUSSION THREAD

**Discussion by amano-kenji** [netblue30/firejail#5268] [Q&A] [answered]
Can it?

**Comment by rusty-snake** [comment, marked as answer]
What du you mean with "spyware in PDF files"?

**Reply by amano-kenji** [reply]
A few examples.

* https://clario.co/blog/can-pdf-contain-virus/
* https://security.stackexchange.com/questions/64052/can-a-pdf-file-contain-a-virus

**Reply by rusty-snake** [reply]
Use a secure pdf reader and you are protected. That's the best thing you can do, secure software following KISS principle without over complexity or anti-security features. You can run such a pdf viewer then in firejail as an additional defense line.

If you have to use a "pdf executor" you can use firejail to mitigate the risk.
